transformer_engine/pytorch/attention.py

```diff
@@ -802,6 +867,17 @@ def get_attention_backend(
 
     # All available backends
     available_backends = [use_flash_attention, use_fused_attention, use_unfused_attention]
+    # if not use_fused_attention and use_flash_attention and not _flash_attn_is_installed:
+    if use_flash_attention and not _flash_attn_is_installed:
```

Comment:
This condition does not do the same thing as the comment says as it does not check if use_fused_attention is false.